### DISCUSSION TITLE
How to show Styles by default in InspectorControls

### DISCUSSION THREAD

**Discussion by QuietNoise** [WordPress/gutenberg#50205] [Q&A]
Hi,
When I add custom controls to InspectorControls (Sidebar) of core blocks using higher order component approach these controls show up in a newly added Settings panel rather than in Styles panel.
The same applies to many core blocks where controls are split into Setting and Styles panel with Settings panel being shown first.
Is there a way for me to enforce Styles panel being shown first rather than Settings?
Pretty much all typical user actions (like formatting etc) are happening in the Styles panel and having to click every single time someone needs to change color or something is very inconvenient.
Alternatively is there a way for me to add custom controls to Styles panel rather than Settings panel?

![image](https://user-images.githubusercontent.com/3076187/235328636-a98ce18b-c7f0-4960-af60-f9a766d3dd12.png)

Plain WordPress 6.2.

**Comment by QuietNoise** [comment]
I've figured out answer to my second question. I can place custom controls into Styles panel using group property i.e. 
`<InspectorControls group="styles">`
source: [Introduction of Block Inspector Tabs](https://make.wordpress.org/core/2023/03/07/introduction-of-block-inspector-tabs/)

I still would like to know whether it is possible to reorder the tabs so the Styles show up first and open.

**Comment by talldan** [comment]
You can hide the tabs completely for a block type (there's an example in the comments of the post you linked), but I don't think you can reorder them. The concern is that this would introduce too much inconsistency in the user experience.
